<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReviewWindow</class>
 <widget class="QMainWindow" name="ReviewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Write a Review</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_rating">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rating:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_rating">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1 - Poor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 - Fair</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 - Good</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 - Very Good</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 - Excellent</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_review">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>350</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_submit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_submit_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
